<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GridNetConfigPage</class>
 <widget class="QFrame" name="GridNetConfigPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item>
    <widget class="QGroupBox" name="grid_group">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Show background decoration</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="3" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Far style</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="10" column="2" colspan="2">
       <widget class="QCheckBox" name="show_ruler">
        <property name="text">
         <string>Show ruler</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="2">
       <widget class="QComboBox" name="style1_cbx">
        <item>
         <property name="text">
          <string>Invisible</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dots</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dotted lines</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Light dotted lines</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dotted lines, ten dots per division </string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Crosses</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Lines</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Lines with ticks, ten ticks per division</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Checkerboard</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="4">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="3">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0" colspan="5">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="8" column="2">
       <widget class="QPushButton" name="grid_axis_color_pb">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Grid   </string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="5">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="grid_net_color_pb">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Ruler</string>
        </property>
       </widget>
      </item>
      <item row="11" column="2">
       <widget class="QPushButton" name="grid_ruler_color_pb">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Axis</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="grid_grid_color_pb">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="Close style">
        <property name="text">
         <string>Close style</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Color (all)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Color</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Style</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="7" column="2" colspan="2">
       <widget class="QComboBox" name="style0_cbx">
        <item>
         <property name="text">
          <string>Invisible</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dots</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dotted lines</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Light dotted lines</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dotted lines, ten dots per division </string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Crosses</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Lines</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Lines with ticks, ten ticks per division</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Color</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Color</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="2" colspan="2">
       <widget class="QComboBox" name="style2_cbx">
        <item>
         <property name="text">
          <string>Invisible</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dots</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dotted lines</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Light dotted lines</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dotted lines, ten dots per division </string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Crosses</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Lines</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Lines with ticks, ten ticks per division</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Checkerboard</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="9" column="0" colspan="5">
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QSpinBox" name="grid_density_sb">
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Min. spacing</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="3" colspan="2">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>(Font height units)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
